CHANGELOG.rst

```diff
@@ -5,6 +5,11 @@ Changelog for django-agnocomplete
 master (unreleased)
 ===================
 
+(nothing yet)
```

Comment:
you can remove the "master unreleased" and (nothing yet) bit, the latest title of this document should be the current release (0.7.0rc1)